src/components/CopyableText.tsx

```diff
@@ -46,15 +44,10 @@ const CopyableText: React.FC<
       return;
     }
     toast({
+      status: "success",
+      description: toastMessage,
       id: TOAST_ID,
-      render: () => (
-        <ToastBody
-          message={toastMessage}
-          onClose={() => {
-            toast.close(TOAST_ID);
-          }}
-        />
-      ),
+      isClosable: true,
```

Comment:
👍

Reply:
though on the screenshots the cross has be centered, and might be also smaller than it has to be
the checkmark is filled with black, but should have transparent background

Reply:
I checked with Bahador & using the chakra confirmation toast is OK for this case.

Ideally we would like to switch to an address pill here - I've created a ticket for that.
https://app.asana.com/0/1205770721172203/1206772726685952/f

And only have CopyableText on the AddressBook page.